onnxruntime/python/tools/quantization/evaluate.py

```diff
@@ -267,3 +269,58 @@ def predict(self):
                 image_id = image_id_batch[i][batch_i]
                 self.set_bbox_prediction(bboxes, scores, image_height, image_width, image_id)
 
+class TRTResNet50Evaluator: 
```

Comment:
I'm not sure I like the direction of the classes in this evaluate module.
the naming refer to specific models and runtimes  (YoloV3, Resnet , TRT)
It seems better to use abstractions that can be applied towards classes of models  (object detection, image classification etc.)

I mean, top-K can be applied to all classification. not just for Resnet50 and TRT.